lib/include/jxl/codestream_header.h

```diff
@@ -232,6 +246,11 @@ typedef struct {
    */
   JxlAnimationHeader animation;
 
+  /** Dimensions of the intrinsic size, only used if have_intrinsic_size is
+   * JXL_TRUE.
+   */
+  JxlIntrinsicSizeHeader intrinsic_size;
+
   /** Padding for forwards-compatibility, in case more fields are exposed
    * in a future version of the library.
    */
```

Comment:
Padding should be adjusted so the struct size remains the same. I think I set it to 108 because I was anticipating 8 bytes for the intrinsic dimensions :)

Reply:
Ok, adjusted it, using 12 bytes, because of the boolean.